api-client/src/errors/NotFoundError.ts

```diff
@@ -1,6 +1,6 @@
 import { LocalNewsType, TuNewsType } from '../routes'
 
-type NotFoundType = 'category' | 'event' | 'poi' | 'offer' | 'disclaimer' | TuNewsType | LocalNewsType | 'route'
+type NotFoundType = 'city' | 'category' | 'event' | 'poi' | 'offer' | 'disclaimer' | TuNewsType | LocalNewsType | 'route'
```

Comment:
Why don't we have 'language' here also?

Reply:
I guess because we treat `LanguageNotAvailable` differently.